src/main/resources/db/migration/V1_23__type.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE type
+(
```

Comment:
Still not sure the name of the table is ok - not currently a reserved word, so ok for now